environments/echis/public.yml

```diff
@@ -0,0 +1,77 @@
+internal_domain_name: moh.gov.et
+
+DATADOG_ENABLED: False
+DATADOG_INTEGRATIONS_ENABLED: False
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_memory: '4096m'
+elasticsearch_cluster_name: 'echis-es'
+elastcsearch_backup_days: 3
+
+nofile_limit: 65536
+ssh_allow_password: True
+
+backup_blobdb: False
+backup_postgres: False
+backup_es: False
+backup_couch: False
+postgres_s3: False
+blobdb_s3: False
+couch_s3: False
+
+KSPLICE_ACTIVE: no
+
+shared_drive_enabled: False
+
+couchdb2:
+  username: "{{ localsettings_private.COUCH_USERNAME }}"
+  password: "{{ localsettings_private.COUCH_PASSWORD }}"
+
+couchdb_cluster_settings:
+  q: 8
+  r: 2
+  w: 2
+  n: 2
+
+couch_dbs:
+  default:
+    host: "{{ groups.couchdb2_proxy[0] }}"
+    port: "{{ couchdb2_proxy_port }}"
+    name: commcarehq
+    username: "{{ localsettings_private.COUCH_USERNAME }}"
+    password: "{{ localsettings_private.COUCH_PASSWORD }}"
+    is_https: False
+
+localsettings:
+  ALLOWED_HOSTS:
+    - echis.moh.gov.et  # TODO
+    - 172.19.0.160
+    - 172.19.0.161
+    - 172.19.3.40
+    - localhost
+    - 127.0.0.1
+  CELERY_PERIODIC_QUEUE: 'celery_periodic'
+  CELERY_REMINDER_CASE_UPDATE_QUEUE: 'reminder_case_update_queue'
+  CELERY_REMINDER_RULE_QUEUE: 'reminder_rule_queue'
+  CELERY_REPEAT_RECORD_QUEUE: 'repeat_record_queue'
```

Comment:
I'm pretty sure these are just repeats of the defaults in commcare-hq/settings.py. Looks like I could have done more cleanup after some of the celery changes I made a while ago.